### PR TITLE
Added info in Security basics

### PR DESCRIPTION
**Pull request by lynnlangit**
added detail about IAM objects
added detail and link to IAM policy simulator tool

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -528,7 +528,7 @@ We cover security basics first, since configuring user accounts is something you
 -	📒 IAM [Homepage](https://aws.amazon.com/iam/) ∙ [User guide](https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html) ∙ [FAQ](https://aws.amazon.com/iam/faqs/)
 -	The [AWS Security Blog](https://blogs.aws.amazon.com/security) is one of the best sources of news and information on AWS security.
 -	**IAM** is the service you use to manage accounts and permissioning for AWS.
--	Managing security and access control with AWS is critical, so every AWS administrator needs to use and understand IAM, at least at a basic level.
+-	Managing security and access control with AWS is critical, so every AWS administrator needs to use and understand IAM, at least at a basic level.  IAM objects include Users, Groups (containers for users and their permissions), Roles(container for permissions assigned to AWS service instances) and Policies (permissions).
```
Comment (jlevy) [inline README.md]: A few suggested changes. Let's make this a new bullet, too:

> - [**IAM identities**](https://docs.aws.amazon.com/IAM/latest/UserGuide/id.html) include **users** (people or services that are using AWS), **groups** (containers for sets of Users and their permissions), and **roles** (containers for permissions assigned to AWS service instances). [**Permissions**](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_permissions.html) for these identities are governed by [**policies**](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies.html).

(Fixes a few typos and trying match AWS capitalization conventions.)

See what you think.

File: README.md [hunk 2]
```diff
@@ -539,6 +539,7 @@ We cover security basics first, since configuring user accounts is something you
 -	At the beginning, IAM policy may be very simple, but for large systems, it will grow in complexity, and need to be managed with care.
 	-	🔹Make sure one person (perhaps with a backup) in your organization is formally assigned ownership of managing IAM policies, make sure every administrator works with that person to have changes reviewed. This goes a long way to avoiding accidental and serious misconfigurations.
 -	It is best to give each user or service the minimum privileges needed to perform their duties. This is the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege), one of the foundations of good security. Organize all IAM users and groups according to levels of access they need.
+-	You can verify policy permissions via the AWS IAM [policy simulator] tool(https://policysim.aws.amazon.com/home/index.jsp). This is particularly useful if you write custom policies.
```
Comment (jlevy) [inline README.md]: > You can **test IAM policies** via the AWS IAM [**policy simulator tool**](https://policysim.aws.amazon.com/home/index.jsp). This is particularly useful if you write custom policies.

(fix link, boldface key concepts)

File: README.md (hunk 1 above)
Comment (ThanosBaskous) [inline README.md]: +2 to suggestion. Additional style note: shouldn't have 2 spaces before new sentence.

**Comment by ThanosBaskous** [comment]
I agree with @jlevy's feedback so far but otherwise LGTM - thanks @lynnlangit!

**Comment by lynnlangit** [comment]
updated per requests

**Comment by ThanosBaskous** [comment]
Thanks @lynnlangit!

**Pull request merged**